server/src/dtos/person.dto.ts

```diff
@@ -181,6 +181,52 @@ export class PeopleResponseDto {
   hasNextPage?: boolean;
 }
 
+export class CreateFaceDto {
+  @ValidateUUID()
+  personId!: string;
+
+  @ValidateUUID()
+  assetId!: string;
+
+  @ApiProperty({ type: 'integer' })
+  @IsNotEmpty()
+  @IsNumber()
+  imageWidth!: number;
+
+  @ApiProperty({ type: 'integer' })
+  @IsNotEmpty()
+  @IsNumber()
+  imageHeight!: number;
+
+  @ApiProperty({ type: 'integer' })
+  @IsNotEmpty()
+  @IsNumber()
+  boundingBoxX1!: number;
+
+  @ApiProperty({ type: 'integer' })
+  @IsNotEmpty()
+  @IsNumber()
+  boundingBoxX2!: number;
+
+  @ApiProperty({ type: 'integer' })
+  @IsNotEmpty()
+  @IsNumber()
+  boundingBoxY1!: number;
+
+  @ApiProperty({ type: 'integer' })
+  @IsNotEmpty()
+  @IsNumber()
+  boundingBoxY2!: number;
+}
+
+export class DeleteFaceDto {
+  @ValidateUUID()
+  assetFaceId!: string;
+
+  @ValidateUUID()
+  personId!: string;
+}
```

Comment:
This shouldn't include `personId` and the access check can infer the person from the `assetFaceId`